hdr/sqlite_modern_cpp/type_wrapper.h

```diff
@@ -31,7 +35,14 @@
 #ifdef MODERN_SQLITE_STD_VARIANT_SUPPORT
 #include <variant>
 #endif
-
+#ifdef MODERN_SQLITE_STRINGVIEW_SUPPORT
+#include <string_view>
+typedef const std::string_view STR_REF;
+typedef const std::u16string_view U16STR_REF;
+#else
+typedef const std::string& STR_REF;
+typedef const std::u16string& U16STR_REF;
+#endif
```

Comment:
The typedef should be in out namespace to avoid introducing names in the global namespace. Also they are no longer macros, so the names should no longer be uppercase.